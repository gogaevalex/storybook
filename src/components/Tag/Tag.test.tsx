import { render } from '@testing-library/react';
import { TestProvider } from '../../test/TestProvider';
import { Tag, ITagProps } from '.';

describe('Tag', () => {
  const defaultProps: ITagProps = {
    content: 'фильмы',
  };
  it('Default render', () => {
    const { container } = render(
      <TestProvider>
        <Tag {...defaultProps} />
      </TestProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
