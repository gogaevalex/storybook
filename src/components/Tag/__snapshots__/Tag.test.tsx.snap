// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag Default render 1`] = `
<div>
  <div
    class="css-mrw1nn"
  >
    <span
      class="css-vsw5yo"
      color="inherit"
    >
      # фильмы
    </span>
  </div>
</div>
`;
